quinn-proto/src/connection.rs

```diff
@@ -2731,10 +2733,6 @@ impl Connection {
         });
     }
 
-    pub fn is_handshaking(&self) -> bool {
```

Comment:
Similarly, this is convenient to determine whether 0-RTT acceptance is defined yet. You could reconstruct that information from events, but it's a lot easier to be able to just look it up.

Reply:
Brought it back.